### ISSUE TITLE
Error on multi-line regexp /.../x

### ISSUE THREAD

**Issue by Tietew** [rubocop-hq/rubocop#7655]
Error occurs with code below:

``` ruby
/(?:
  blah
)/x =~ blah
```

Works:

``` ruby
/(?:single_line)/ =~ blah
```

``` ruby
/
  no_capture
/x =~ blah
```

--------

## Expected behavior

No errors.

## Actual behavior

An error occurred while VariableForce cop

## Steps to reproduce the problem

```
$ ruby -vc test.rb
ruby 2.6.5p114 (2019-10-01 revision 67812) [x86_64-linux]
Syntax OK
```
```
$ bundle exec rubocop test.rb
For .../test/rubocop: configuration from .../test/rubocop/.rubocop.yml
Default configuration from .../gems/rubocop-0.79.0/config/default.yml
Inspecting 1 file
Scanning .../test/rubocop/test.rb
An error occurred while VariableForce cop was inspecting .../test/rubocop/test.rb.
end pattern with unmatched parenthesis: /(?:
/
.../gems/rubocop-0.79.0/lib/rubocop/cop/variable_force.rb:194:in `initialize'
.../gems/rubocop-0.79.0/lib/rubocop/cop/variable_force.rb:194:in `new'
.../gems/rubocop-0.79.0/lib/rubocop/cop/variable_force.rb:194:in `regexp_captured_names'
.../gems/rubocop-0.79.0/lib/rubocop/cop/variable_force.rb:174:in `process_regexp_named_captures'
.../gems/rubocop-0.79.0/lib/rubocop/cop/variable_force.rb:86:in `process_node'
.../gems/rubocop-0.79.0/lib/rubocop/cop/variable_force.rb:103:in `block in process_children'
.../gems/rubocop-0.79.0/lib/rubocop/ast/node.rb:193:in `block in each_child_node'
.../gems/rubocop-0.79.0/lib/rubocop/ast/node.rb:190:in `each'
.../gems/rubocop-0.79.0/lib/rubocop/ast/node.rb:190:in `each_child_node'
.../gems/rubocop-0.79.0/lib/rubocop/cop/variable_force.rb:100:in `process_children'
.../gems/rubocop-0.79.0/lib/rubocop/cop/variable_force.rb:87:in `process_node'
.../gems/rubocop-0.79.0/lib/rubocop/cop/variable_force.rb:80:in `investigate'
.../gems/rubocop-0.79.0/lib/rubocop/cop/commissioner.rb:108:in `block (2 levels) in invoke_custom_processing'
.../gems/rubocop-0.79.0/lib/rubocop/cop/commissioner.rb:136:in `with_cop_error_handling'
.../gems/rubocop-0.79.0/lib/rubocop/cop/commissioner.rb:107:in `block in invoke_custom_processing'
.../gems/rubocop-0.79.0/lib/rubocop/cop/commissioner.rb:104:in `each'
.../gems/rubocop-0.79.0/lib/rubocop/cop/commissioner.rb:104:in `invoke_custom_processing'
.../gems/rubocop-0.79.0/lib/rubocop/cop/commissioner.rb:43:in `investigate'
.../gems/rubocop-0.79.0/lib/rubocop/cop/team.rb:124:in `investigate'
.../gems/rubocop-0.79.0/lib/rubocop/cop/team.rb:112:in `offenses'
.../gems/rubocop-0.79.0/lib/rubocop/cop/team.rb:44:in `inspect_file'
.../gems/rubocop-0.79.0/lib/rubocop/runner.rb:296:in `inspect_file'
.../gems/rubocop-0.79.0/lib/rubocop/runner.rb:244:in `block in do_inspection_loop'
.../gems/rubocop-0.79.0/lib/rubocop/runner.rb:276:in `block in iterate_until_no_changes'
.../gems/rubocop-0.79.0/lib/rubocop/runner.rb:269:in `loop'
.../gems/rubocop-0.79.0/lib/rubocop/runner.rb:269:in `iterate_until_no_changes'
.../gems/rubocop-0.79.0/lib/rubocop/runner.rb:240:in `do_inspection_loop'
.../gems/rubocop-0.79.0/lib/rubocop/runner.rb:119:in `block in file_offenses'
.../gems/rubocop-0.79.0/lib/rubocop/runner.rb:143:in `file_offense_cache'
.../gems/rubocop-0.79.0/lib/rubocop/runner.rb:117:in `file_offenses'
.../gems/rubocop-0.79.0/lib/rubocop/runner.rb:108:in `process_file'
(snip)
.

1 file inspected, no offenses detected

1 error occurred:
An error occurred while VariableForce cop was inspecting .../test/rubocop/test.rb.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop-hq/rubocop/issues

Mention the following information in the issue report:
0.79.0 (using Parser 2.7.0.2, running on ruby 2.6.5 x86_64-linux)
Finished in 0.10937485797330737 seconds
```

## RuboCop version

```
$ [bundle exec] rubocop -V
0.79.0 (using Parser 2.7.0.2, running on ruby 2.6.5 x86_64-linux)
```
